extension/options/optionsView.jsx

```diff
@@ -164,12 +164,14 @@ this.optionsView = (function() {
     const isMac = window.navigator.platform.match(/Mac/i);
     let modifier1, modifier2, placeholder;
     if (isMac) {
-      modifier1 = "Command Alt MacCtrl";
-      modifier2 = "Command Alt MacCtrl Shift";
+      modifier1 =
+        "<code>Command</code>, <code>Alt</code>,  <code>MacCtrl</code>";
+      modifier2 =
+        "<code>Command</code>, <code>Alt</code>, <code>MacCtrl</code>, <code>Shift</code>";
       placeholder = "Command+Period";
     } else {
-      modifier1 = "Ctrl Alt";
-      modifier2 = "Ctrl Alt Shift";
+      modifier1 = "<code>Ctrl</code>, <code>Alt</code>";
+      modifier2 = "<code>Ctrl</code>, <code>Alt</code>, <code>Shift</code>";
```

Comment:
Could these just be React.Fragments? That would avoid the dangerouslySetInnerHTML stuff